.travis.yml

```diff
@@ -9,16 +9,16 @@ branches:
 
 env:
   # default is stable repo
-  - DISTRO=ubuntu-16 LICENSE='BWC_LICENSE_ENTERPRISE'
-  - DISTRO=ubuntu-18 LICENSE='BWC_LICENSE_ENTERPRISE'
-  - DISTRO=centos-7 LICENSE='BWC_LICENSE_ENTERPRISE'
-  - DISTRO=centos-8 LICENSE='BWC_LICENSE_ENTERPRISE'
+  - DISTRO=ubuntu-16
+  - DISTRO=ubuntu-18
+  - DISTRO=centos-7
+  - DISTRO=centos-8
 
   # StackStorm 'unstable' repo check
-  - DISTRO=ubuntu-16 ST2_REPO=unstable EWC_REPO=enterprise-unstable LICENSE='BWC_LICENSE_ENTERPRISE_UNSTABLE'
-  - DISTRO=ubuntu-18 ST2_REPO=unstable EWC_REPO=enterprise-unstable LICENSE='BWC_LICENSE_ENTERPRISE_UNSTABLE'
-  - DISTRO=centos-7 ST2_REPO=unstable EWC_REPO=enterprise-unstable LICENSE='BWC_LICENSE_ENTERPRISE_UNSTABLE'
-  - DISTRO=centos-8 ST2_REPO=unstable EWC_REPO=enterprise-unstable LICENSE='BWC_LICENSE_ENTERPRISE_UNSTABLE'
```

Comment:
:+1: 
There's also missing https://github.com/StackStorm/ansible-st2/blob/master/.kitchen.yml file where we could remove the  license-related code.

Reply:
Done, the hidden file slipped the grep. :smile: